packages/home/sample_data_tab_content/src/demo_env_panel.tsx

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { useEffect, useRef, useState } from 'react';
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiText,
+  EuiImage,
+  useEuiTheme,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+const title = i18n.translate('homePackages.demoEnvironmentPanel.welcomeTitle', {
+  defaultMessage: 'Explore our live demo environment',
+});
+
+const message = i18n.translate('homePackages.demoEnvironmentPanel.welcomeMessage', {
+  defaultMessage:
+    "Browse our demo environment and explore the many ways you can gain insights into your own data using Elastic. With its real-world data for search, observability, and security, it's just a few clicks before you find something that you can leverage for your own use cases.",
```

Comment:
This text doesn't match what [we settled on in the issue](https://github.com/elastic/kibana/issues/135076#issuecomment-1184915565). 
```suggestion
    "Browse real-world data in a demo environment where you can explore search, observability, and security use cases like yours.",
```